resource_virtual_machine.go

```diff
@@ -228,6 +233,53 @@ func resourceVirtualMachineCreate(d *schema.ResourceData, meta interface{}) erro
 		confSpec.MemoryMB = int64(d.Get("memory").(int))
 	}
 
+	var devices object.VirtualDeviceList
+
+	for _, diskValue := range d.Get("disks").([]interface{}) {
+		if disk, ok := diskValue.(map[string]interface{}); ok {
+			diskDatastoreName := disk["datastore"].(string)
+
+			datastore, err := finder.Datastore(ctx, fmt.Sprintf("/%v/datastore/%v", dc_name, diskDatastoreName))
+			if err != nil {
+				return fmt.Errorf("Failed to find datastore \"%s\": %v", diskDatastoreName, err)
+			}
+
+			datastoreRef := datastore.Reference()
+
+			controllerDevice, err := devices.CreateSCSIController("")
```

Comment:
AFAIK yes, as it's usual IDE/SATA/SCSI controller and supports many disks/cdroms/etc.
@mkuzmin don't you remember where's groovy code we created >year ago to copy/attach disks in?